docs/content/guide/i18n.ngdoc

```diff
@@ -249,3 +355,27 @@ This is taken from the [plunker example](http://plnkr.co/edit/QBVRQ70dvKZDWmHW9R
     other {You ({{sender.name}}) gave {{recipients.length}} people gifts. }
 }}
 ```
+
+### Differences from the ICU MessageFormat syntax
+
+This section is useful to you if you're already familiar with the ICU MessageFormat syntax.
+
+This syntax extension, while based on MessageFormat, has been designed to be backwards compatible
+with existing AngularJS interpolation expressions.  The key rule is simply this: **All
+interpolations are done inside double curlies.** The top level comma operator after an expression
+inside the double curlies causes MessageFormat extensions to be recognized.  Such a top level comma
+is otherwise illegal in an Angular expression and is used by MessageFormat to specify the function
+(such as plural/select) and it's related syntax.
+
+To understand the extension, take a look at the ICU MessageFormat syntax as specified by the ICU
+documentation.  Anywhere in that MessageFormat that you have regular message text and you want to
+substitute an expression, just put it in double curlies instead of single curlies that MessageFormat
+dictates.  This has a huge advantage.  **You are no longer limited to simple identifiers for
+substitutions**.  Because you are using double curlies, you can stick in any arbitrary interpolation
+syntax there, including nesting more MessageFormat expressions!  Some examples will make this clear.
+In the following example, I will only be showing you the AngularJS syntax.
```

Comment:
Removed the sentence.  This paragraph occurred a lot earlier in the previous version and it doesn't make sense anymore.